### ISSUE TITLE
The question about KLD loss 

### ISSUE THREAD

**Issue by haoopan** [Cadene/block.bootstrap.pytorch#11]
Excuse me again! 
  When I use the KLDivLoss as the criterion in ./block/models/criterions/kl_divergence.py rather than crossentropy, I get the  loss is 0 and the acc is worse , as following picture shows:
![image](https://user-images.githubusercontent.com/35628413/63181798-8a985a80-c083-11e9-8975-8b520c841d72.png)
  But in paper MFH ,KLD loss is better than CrossEntropy, So, how could this be? Thanks!
